### ISSUE TITLE
Error once skill linked

### ISSUE THREAD

**Issue by pceasar** [keatontaylor/alexa-actions#169]
Hi, not sure if I'm the only one with this issue, when I follow Keaton's directions to create the skill and even link it in my Alexa, once I run the test of "open actionable notification" I get "This is a test of the alexa actionable notification skill. Did it work?" whether I say "yes" or "no" - I get this error "Error 401 It looks like I am unauthorized to reach home assistant, please check your account linking or your long lived access token and try again" 

I have nabu casa and I was able to link within the Alexa app...Any thoughts? I hope this is the right place to submit this post.

**Comment by DEADSEC-SECURITY** [comment]
Please follow the updated repository wiki. The youtube videos are not updated to the latest version

**Comment by pceasar** [comment]
Ye

> Also did you login to HA via the alexa app? Have you tried using tokens?

Yes I did do the linking in Alexa and that's how it linked in my app. I can try with a token and remove linking

**Comment by pceasar** [comment]
> Please follow the updated repository wiki. The youtube videos are not updated to the latest version

Yes I followed the repository but it did not mention linking I don't believe, I will re-follow it a 3rd time for good measure. Thank you so much for responding me.
